plugins/toolkit/core/src/software/aws/toolkits/core/ToolkitClientManager.kt

```diff
@@ -155,6 +158,8 @@ abstract class ToolkitClientManager {
                     // async clients use CRT, and keeps trying to shut down our apache client even though it doesn't respect our client settings
                     // so only set this for sync clients
                     httpClient(sdkHttpClient())
+                } else if (this is SdkAsyncClientBuilder<*, *>) {
+                    httpClient(sdkAsyncHttpClient())
```

Comment:
Philipp suggested we have a new netty client each time since the proxy configuration is static

Reply:
Done